app/aggregates/deciding/overall_result_calculator.rb

```diff
@@ -41,6 +42,12 @@ def failed_means?
       decision.means&.result == 'failed'
     end
 
+    def ineligible?
+      return false unless decision.assessment_rules == 'crown_court'
```

Comment:
☝️ this is what all the fuss was about. We can now treat things assessed under Crown Court Trial rules differently.